docs/release/1.25/release-note_1.25.0.md

```diff
@@ -0,0 +1,17 @@
+# Release Note 1.25.0
+
+## ONE Runtime
+
+- Support ubuntu 20.04
+
+### CPU Backend Operation
+- CPU backend supports per-channel hybrid quantization of int8 type weight and float activation. (TFLite's dynamic range quantization)
+
+### On-device Quantization
+- _onert_ supports new experimental API for on-device quantization.
+- As a 1st step, _onert_ supports per-channel hybrid quantization of int8/int16 type weight and float activation.
```

Comment:
```suggestion
- As the 1st step, _onert_ supports per-channel hybrid quantization of int8/int16 type weight and float activation.
```

Reply:
:+1:

Reply:
@hseok-oh As I said offline, I found "as a 1st step" is not incorrect. It is just an idiom. Now, I am okay for both "a" and "the" in the sentence above :)